app/assets/javascripts/admin/spree/taxons/taxonomy.js.coffee

```diff
@@ -40,8 +40,8 @@ handle_rename = (e, data) ->
   node = data.rslt.obj
   name = data.rslt.new_name
 
-  url = Spree.url(base_url).clone()
-  url.pathname = url.pathname + '/' + node.attr("id")
+  url =  $.extend(true, {}, base_url)
```

Comment:
I'd vote to avoid jQuery every time this is possible.
Here, I think `$.extend()` could easily be replaced by `Object.assign()`.

```suggestion
  url = Object.assign(base_url, {})
```